admin/partials/setup/setup-section-display.php

```diff
@@ -4,15 +4,15 @@
  * This file is used to setup a settings section
  *
  * @link       https://www.enzuzo.com
- * @since      1.0.0
+ * @since      1.0.1
  *
  * @package    Enzuzo_Cookie_Consent
  * @subpackage Enzuzo_Cookie_Consent/admin/partials
  */
 ?>
-<?php echo _e('<p>Find out more about Enzuzo Cookie Consent <a target="_blank" href="https://www.enzuzo.com/consent-management-software">here</a>.</p>'); ?>
-<?php echo _e('<p>Don\'t have an account yet?  Create a new account to customize <a target="_blank" href="https://app.enzuzo.com/onboarding">here</a>.</p>'); ?>
-<?php echo _e('<p>Already have an account?  Login to your account <a target="_blank" href="https://app.enzuzo.com">here</a>.</p>'); ?>
-<?php echo _e('<h3>Where to find UUID</h3>'); ?>
-<?php echo _e('<p>You can find the UUID (looks like aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee) or JavaScript snippet from this <a target="_blank" href="https://app.enzuzo.com/settings/cookiebar/add-to-website">path</a> in the Enzuzo Admin Dashboard.'); ?>
+<?php echo '<p>'.esc_html('Find out more about Enzuzo Cookie Consent ').'<a target="_blank" href="https://www.enzuzo.com/consent-management-software">'.esc_html('here').'</a>.</p>' ?>
+<?php echo '<p>'.esc_html('Don\'t have an account yet?  Create a new account to customize ').'<a target="_blank" href="https://app.enzuzo.com/onboarding">'.esc_html('here').'</a>.</p>' ?>
+<?php echo '<p>'.esc_html('Already have an account?  Login to your account ').'<a target="_blank" href="https://app.enzuzo.com">'.esc_html('here').'</a>.</p>' ?>
+<?php echo '<h3>'.esc_html('Where to find UUID').'</h3>' ?>
+<?php echo '<p>'.esc_html('You can find the UUID (looks like aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee) or JavaScript snippet from this ').'<a target="_blank" href="https://app.enzuzo.com/settings/cookiebar/add-to-website">'.esc_html('path').'</a> in the Enzuzo Admin Dashboard.' ?>
```

Comment:
Change this text as well to be installation code snippet or UUID

Feel free to refactor the text - seems like it's repeated a few places already.  Not a big deal.